qa-tests-backend/test-images/trigger-policy-violations/most/Dockerfile

```diff
@@ -8,9 +8,14 @@ WORKDIR /
 # RUN echo "a secret" > /run/secrets/asecret
 # VOLUME /run/secrets
 
+RUN yum update -y
+
 # For: "Shell Spawned by Java Application"
 RUN yum -y install csh
 
+# For: trigger-violations.sh to trigger systemctl violation
+RUN yum update -y && yum install systemd -y
```

Comment:
@ksurabhi91 , Should we also add a line to install `systemctl` directly, too?

Reply:
No. I tested this when making https://github.com/stackrox/stackrox/pull/12211

```
❯ docker run --rm -it --platform linux/amd64 quay.io/centos/centos:stream9

[root@95cc7540451e /]# yum install which -y
...
Complete!

[root@95cc7540451e /]# which systemctl
which: no systemctl in (/root/.local/bin:/root/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sb

[root@95cc7540451e /]# yum update -y && yum install systemd -y
...
Complete!

[root@95cc7540451e /]# which systemctl
/usr/bin/systemctl
```